upstream-sha256: 77e49e86d1d72a9452b48c4c8ce97de26200b6ff0829b9874f9414397383f2bd
actual-sha256: 688d606885378a0fc386a0f38710f28e3e6043aecb9b795103b5a5ec00605872
--- upstream.expected
+++ tolk.actual
@@ -6,7 +6,7 @@
     int alu1 = (Ridx0<<2);
     float4 val0 = (*((float4*)((data1_256+alu1))));
     float4 val1 = (*((float4*)((data2_256+alu1))));
-    *(buf0+0) = ((*(buf0+0))+(val0[0]*val1[0])+(val0[1]*val1[1])+(val0[2]*val1[2])+(val0[3]*val1[3]));
+    *(buf0+0) = ((*(buf0+0))+((val0[0]*val1[0])+(val0[1]*val1[1])+(val0[2]*val1[2])+(val0[3]*val1[3])));
   }
   *(data0_1+0) = (*(buf0+0));
 }
